src/background/services/background.ts

```diff
@@ -108,7 +108,9 @@ export class Background {
 
   bindTabHandlers() {
     this.browser.tabs.onRemoved.addListener(this.tabEvents.clearTabSessions)
-    this.browser.tabs.onUpdated.addListener(this.tabEvents.clearTabSessions)
+    this.browser.tabs.onUpdated.addListener(this.tabEvents.clearTabSessions, {
+      properties: ['url', 'discarded', 'status', 'isArticle']
+    })
```

Comment:
In Chrome:

<img width="628" alt="image" src="https://github.com/user-attachments/assets/bd62a42f-a24f-493d-bb00-b53692af846e">